src/configure/configure.ts

```diff
@@ -186,8 +188,9 @@ class Orchestrator {
                 let extendedPipelineTemplate = await templateHelper.getTemplateParameteres(this.inputs.azureSession, (this.inputs.pipelineConfiguration.template as RemotePipelineTemplate));
                 let context: { [key: string]: any } = {};
                 context['subscriptionId'] = this.inputs.subscriptionId;
+                context['rightClickScenario'] = rightClickScenario;
```

Comment:
create it as private member and set the context values, wherever theses fields are set. Like subcriptionId at the place where subscriptionId is fetched.

Reply:
Updated